Source/WebService/Blackjack.WebService/Controllers/UserController.cs

```diff
@@ -37,4 +37,13 @@ public async Task<ActionResult<UserBM>> GetUser([FromQuery] string username)
         UserBM result = user == null ? null : _mapper.Map<UserBM>(user);
         return Ok(result);
     }
+
+    [HttpPost]
+    public async Task<ActionResult<UserBM>> UpdateUser([FromBody] UserBM user)
+    {
+        UserBE userBE = _mapper.Map<UserBE>(user);
+        UserBE updatedUserBE = await _userAdapter.UpdateUser(userBE);
+        UserBM updatedUserBM = updatedUserBE == null ? null : _mapper.Map<UserBM>(updatedUserBE);
```

Comment:
This will never be null b/c you correctly throw a request-killing exception if the user cannot be found during the update process.